modules/data.land/R/load_veg.R

```diff
@@ -0,0 +1,107 @@
+##' Function uses load_data{benchmark} to get veg data
+##' @name load_veg
+##' @title load_veg
+##' @export
+##' @author Istem Fer
+load_veg <- function(new_site, start_date, end_date, 
+                     source_id, source, icmeta = NULL, format_name = NULL, 
+                     machine_host, dbparms, outfolder, overwrite = FALSE, ...){
+  
+  
+  bety <- dplyr::src_postgres(dbname   = dbparms$bety$dbname, 
+                              host     = dbparms$bety$host, 
+                              user     = dbparms$bety$user, 
+                              password = dbparms$bety$password)
```

Comment:
This snippet of code is being cut-and-pasted everywhere in PEcAn -- it really should be a simple function like db.open (indeed, db.open should probably be deprecated and replaced globally by this new function).